databricks_cli/sdk/service.py

```diff
@@ -997,29 +997,42 @@ def deploy(self, pipeline_id=None, id=None, name=None, storage=None, configurati
                 raise TypeError('Expected databricks.Filters() or dict for field filters')
         if allow_duplicate_names is not None:
             _data['allow_duplicate_names'] = allow_duplicate_names
-        return self.client.perform_query('PUT', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data, headers=headers)
-    
+        return self.client.perform_query('PUT', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data,
+                                         headers=headers)
+
     def delete(self, pipeline_id=None, headers=None):
         _data = {}
-    
-        return self.client.perform_query('DELETE', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data, headers=headers)
-    
+
+        return self.client.perform_query('DELETE', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id),
+                                         data=_data, headers=headers)
+
     def get(self, pipeline_id=None, headers=None):
         _data = {}
-    
-        return self.client.perform_query('GET', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data, headers=headers)
-    
+
+        return self.client.perform_query('GET', '/pipelines/{pipeline_id}'.format(pipeline_id=pipeline_id), data=_data,
+                                         headers=headers)
+
+    def list(self, pagination=None, headers=None):
+        _data = {}
+        if pagination is not None:
+            if not isinstance(pagination, dict):
+                raise TypeError('Expected databricks.Pagination() or dict for field pagination')
+        return self.client.perform_query('GET', '/pipelines', data=_data, headers=headers)
```

Comment:
Hmm, is this what gets auto-generated? Probably best to leave the line adding pagination into _data here, since we don't call it anyway and we don't want to confuse anyone else that regenerates and updates this file.